pmm-tests/check_client_upgrade.py

```diff
@@ -0,0 +1,104 @@
+import subprocess
+import sys
+
+arguments = sys.argv
+print(arguments)
+
+containers = subprocess.run(["docker", "ps", "-a"], capture_output=True, text=True).stdout.splitlines()
+
+psContainerName = ""
+pgsqlContainerName = ""
+firstMongoReplica = ""
+secondMongoReplica = ""
+thirdMongoReplica = ""
+errors = []
+
+for i in range(len(containers)):
+  if "ps_pmm_" in containers[i]:
+      psContainerName = containers[i][containers[i].index("ps_pmm"):]
+  elif "pgsql_pgss_pmm" in containers[i]:
+      pgsqlContainerName = containers[i][containers[i].index("pgsql_pgss_pmm"):]
+  elif "rs101" in containers[i]:
+    firstMongoReplica = containers[i][containers[i].index("rs101"):]
+  elif "rs102" in containers[i]:
+    secondMongoReplica = containers[i][containers[i].index("rs102"):]
+  elif "rs103" in containers[i]:
+    thirdMongoReplica = containers[i][containers[i].index("rs103"):]
+
+psContainerStatus = subprocess.run(["docker", "exec", psContainerName, "pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+pgContainerStatus = subprocess.run(["docker", "exec", pgsqlContainerName, "pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+firstMongoReplicaStatus = subprocess.run(["docker", "exec", firstMongoReplica, "pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+secondMongoReplicaStatus = subprocess.run(["docker", "exec", secondMongoReplica, "pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+thirdMongoReplicaStatus = subprocess.run(["docker", "exec", thirdMongoReplica, "pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+localClientStatus = subprocess.run(["pmm-admin", "status"], capture_output=True, text=True).stdout.splitlines()
+
+if "Waiting" in psContainerStatus or "Done" in psContainerStatus or "Unknown" in psContainerStatus or "Initialization Error" in psContainerStatus or "Stopping" in psContainerStatus:
```

Comment:
Use positive test check for status running, execute only when container is running specific database.